src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java

```diff
@@ -212,9 +215,23 @@
   private static final String QUOTE_SQL_IDENTIFIERS_DISPLAY =
       JdbcSourceConnectorConfig.QUOTE_SQL_IDENTIFIERS_DISPLAY;
 
+  public static final String TABLE_TYPES_CONFIG = "table.types";
+  private static final String TABLE_TYPES_DISPLAY = "Table Types";
+  public static final String TABLE_TYPES_DEFAULT = TableType.TABLE.toString();
+  private static final String TABLE_TYPES_DOC =
+      "The comma-separated types of database tables to which the sink connector can write. "
+      + "By default this is ``" + TableType.TABLE + "``, but any combination of ``"
+      + TableType.TABLE + "`` and ``" + TableType.VIEW + "`` is allowed. Not all databases "
+      + "support writing to views, and when they do the the sink connector will fail if the "
+      + "view definition does not match the records' schemas (regardless of ``"
+      + AUTO_EVOLVE + "``).";
+
   private static final EnumRecommender QUOTE_METHOD_RECOMMENDER =
       EnumRecommender.in(QuoteMethod.values());
 
+  private static final EnumRecommender TABLE_TYPES_RECOMMENDER =
+      EnumRecommender.in(TableType.values());
```

Comment:
want to add this to the identical source config too?

Reply:
Source connector supports more types. Might need a different `values`.

Reply:
@Cyril-Engels is correct. Plus, I want to *not* change the source connector in any way.